pkg/server/api.go

```diff
@@ -46,6 +47,10 @@ func (a *APIServer) Serve() {
 	mcs := &http.Server{
 		Addr:    fmt.Sprintf(":%v", a.port),
 		Handler: a.handler,
+	// We don't want to allow 1.1 as that's old.  This was flagged in a security audit.
```

Comment:
gofmt is complaining, perhaps indentation issue. Otherwise looks sane.

Reply:
fixed thanks!